README.adoc

```diff
@@ -1,9 +1,37 @@
-= Address Book (Level 3)
+= ORGANice (Organ Transplant Manager)
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
-https://travis-ci.org/AY1920S1-CS2103T-T13-1/main[image:https://travis-ci.org/AY1920S1-CS2103T-T13-1/main.svg?branch=master[Build Status]]
-https://coveralls.io/github/AY1920S1-CS2103T-T13-1/main?branch=master[image:https://coveralls.io/repos/github/AY1920S1-CS2103T-T13-1/main/badge.svg?branch=master[Coverage Status]]
+== Introduction
+
+ORGANice is an organ transplant manager. +
+This desktop application aims to facilitate the matching of organ transplants between patients and organ donors in hospitals. +
+It is written in Java language in an Object-oriented fashion.
+
+== Motivation
+
+It is undeniable that organ transplantation is one of the most life-changing medical breakthroughs in history. +
+For a transplant to be successful, we need to have an organ that matches the receiver. +
+Since time is precious during transplant, we aim to provide an organ matching system to efficiently match organs from donors to the patients who need them.
+
+== Target Users
+
+This application is intended for hospital admin staff such as doctors and nurses.
+
+== Installation
+
+You can download the https://google.com/[jar] file and run it from your desktop
```

Comment:
for jar file use our release link `https://github.com/AY1920S1-CS2103T-T13-1/main/releases`

Reply:
Noted with thanks